src/ApplicationCore/Entities/FraudProtectionApiModels/Response/Enrichments.cs

```diff
@@ -45,6 +45,18 @@ public class DeviceAttributes
         public string UserAgentBrowser { get; set; }
 
         public string UserAgentOperatingSystem { get; set; }
+
+        public string ProxyIp { get; set; }
```

Comment:
How do you know about API updates and where to see the new contracts?

Reply:
PM contacted me.  Worked with someone from a BE team to test these specific fields since we don't typically run proxies.